ci/Jenkinsfiles/build.groovy

```diff
@@ -281,7 +363,7 @@ pipeline {
                 """
                 if (isReferenceBranch) {
                   // When not using jx preview, we need to expose the nuxeo url by hand
-                  url = sh(returnStdout: true, script: "kubectl get svc --namespace ${PREVIEW_NAMESPACE} preview -o go-template='{{index .metadata.annotations \"fabric8.io/exposeUrl\"}}'")
+                  url = sh(returnStdout: true, script: "kubectl get svc --namespace ${PREVIEW_NAMESPACE} preview-webui -o go-template='{{index .metadata.annotations \"fabric8.io/exposeUrl\"}}'")
```

Comment:
```suggestion
                  url = sh(returnStdout: true, script: "kubectl get svc --namespace ${PREVIEW_NAMESPACE} preview-preview -o go-template='{{index .metadata.annotations \"fabric8.io/exposeUrl\"}}'")
```
as we can see in
```kubectl get svc --namespace nuxeo-sample-final1-pr-4 -oname
service/preview-nuxeo
service/preview-preview
```

Reply:
BTW, this is because the chart name deploying the webui templates is still the `preview` one.
Could probably be solved by having a local `webui` chart and the `preview` one depending on it, as it depends on the nuxeo one.